### ISSUE TITLE
Feedback from @hdv@front-end.social

### ISSUE THREAD

**Issue by patak-cat** [elk-zone/elk#195]
Some comments: 

## Accessibility
- [x] the gold color used in a lot of places (like links) doesn't have sufficient contrast with white, should be > 4.5 : 1
- [x] the logo `<img>` doesn't have an `alt` attribute (replace the `aria-label` with `alt`) 
- [x] there are icon buttons with no text content or labels (see https://hidde.blog/naming-things-to-improve-accessibility/)

## Mastodon features
- [ ] can't add language to Toots

## Rendering features
- [x] if I use backticks to render code it shows as three backticks (?)

## Internationalization
- [x] HTML element lacks `lang` attribute
